File: study-ai-all/flutter_app/lib/models/person.dart
class Person {
  final int id;
  final String name;
  final String gender;
  final String birthDate;
  final String? deathDate;
  final String description;

  Person({
    required this.id,
    required this.name,
    required this.gender,
    required this.birthDate,
    this.deathDate,
    required this.description,
  });

  factory Person.fromJson(Map<String, dynamic> json) {
    return Person(
      id: json['id'],
      name: json['name'],
      gender: json['gender'],
      birthDate: json['birthDate'],
      deathDate: json['deathDate'],
      description: json['description'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'gender': gender,
      'birthDate': birthDate,
      'deathDate': deathDate,
      'description': description,
    };
  }
}

File: study-ai-all/spring-boot-backend/src/main/java/com/example/backend/model/Person.java
package com.example.backend.model;

import java.time.LocalDate;

public class Person {
    private Long id;
    private String name;
    private String gender;
    private LocalDate birthDate;
    private LocalDate deathDate;
    private String description;

    public Person() {
    }

    public Person(Long id, String name, String gender, LocalDate birthDate, LocalDate deathDate, String description) {
        this.id = id;
        this.name = name;
        this.gender = gender;
        this.birthDate = birthDate;
        this.deathDate = deathDate;
        this.description = description;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public LocalDate getBirthDate() {
        return birthDate;
    }

    public void setBirthDate(LocalDate birthDate) {
        this.birthDate = birthDate;
    }

    public LocalDate getDeathDate() {
        return deathDate;
    }

    public void setDeathDate(LocalDate deathDate) {
        this.deathDate = deathDate;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}